user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleDetailsComponent.java

```diff
@@ -119,15 +115,13 @@ public SampleDetailsComponent(@Autowired SampleInformationService sampleInformat
     noSamplesRegisteredDisclaimer.setVisible(false);
 
     experimentTabContent.add(sampleGrid, noGroupsDefinedDisclaimer, noSamplesRegisteredDisclaimer);
-
-    TabSheet sampleExperimentTabSheet = new TabSheet();
-    sampleExperimentTabSheet.add(experimentTab, experimentTabContent);
-    sampleExperimentTabSheet.setHeightFull();
-
+    
     Div content = new Div();
     content.addClassName("sample-details-content");
 
-    content.add(buttonAndFieldBar, metadataDownload, sampleExperimentTabSheet);
+    countSpan = new Span();
```

Comment:
Your count span does not have any CSS class. This makes it hard in the future to style it.

Reply:
When, how and what will be re-worked?